redash/handlers/queries.py

```diff
@@ -145,23 +148,16 @@ def get(self):
         ordered_results = order_results(results, fallback=not bool(search_term))
 
         page = request.args.get("page", 1, type=int)
-        page_size = request.args.get("page_size", 25, type=int)
-
-        response = paginate(
-            ordered_results,
-            page=page,
-            page_size=page_size,
-            serializer=QuerySerializer,
-            with_stats=True,
-            with_last_modified_by=False,
-        )
+        per_page = request.args.get("per_page", 25, type=int)
+
+        results = paginate(ordered_results, page=page, per_page=per_page, serializer=QuerySerializer)
```

Comment:
Sorry, I missed this. Could you let me know why you removed `with_stats` and `with_last_modified_by`?

Reply:
restored it back